content/en/tracing/guide/ingestion_sampling_use_cases.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Trace Sampling Use Cases
+kind: guide
+further_reading:
+- link: "/tracing/guide/trace_ingestion_volume_control/"
+  tag: "Guide"
+  text: "How to control ingested volumes"
+---
+
+## Overview
+
+Trace data tends to be repetitive. A problem in your application is rarely identified in a single trace. For high throughput services, particularly for incidents that require your attention, an issue shows symptoms in multiple traces. Consequently, there’s usually no need for you to collect every single trace for a service or endpoint. Datadog APM [ingestion control mechanisms][1] help you keep the visibility that you need to troubleshoot problems while cutting down the noise and managing costs.
```

Comment:
```suggestion
Trace data tends to be repetitive, and a single trace is rarely enough to fully debug an issue. For high throughput services, particularly for incidents that require your attention, crucial data is spread across multiple traces. Consequently, there’s usually no need for you to collect every single trace for a service or endpoint, or every span within a trace. Datadog APM [ingestion control mechanisms][1] provides the visibility that you need to troubleshoot problems while cutting down the noise and managing costs.
```

Reply:
Not sure I agree with this rewrite. The information is not "spread across multiple traces"; it's that a significant problem will show up the same way, repeatedly" in multiple traces, so it doesn't matter which instance of the problem traces you ingest.

Reply:
modified a bit

Reply:
Ah I understand what this section is saying now. I found it confusing at first, and thought it was alluding to multiple traces containing the data.